acl/policy_authorizer_oss.go

```diff
@@ -0,0 +1,30 @@
+// +build !consulent
+
+package acl
+
+// enterprisePolicyAuthorizer stub
+type enterprisePolicyAuthorizer struct{}
+
+func (authz *enterprisePolicyAuthorizer) init(*EnterpriseACLConfig) {
+	// nothing to do
+}
+
+func (authz *enterprisePolicyAuthorizer) enforce(*EnterpriseRule, *EnterpriseAuthorizerContext) EnforcementDecision {
+	return Allow
```

Comment:
Shouldn't this be `Default` for OSS? It feels weird to not in some way have this layer using the default policy action from config.

Reply:
I changed it and added a `defaultIsAllow` func to translate any `Allow` or `Default` response to an `Allow`